fabric-key-binding-api-v1/build.gradle

```diff
@@ -0,0 +1,7 @@
+archivesBaseName = "fabric-key-binding-api-v1"
+version = getSubprojectVersion(project, "1.0.0")
+
+dependencies {
+	compile project(path: ':fabric-api-base', configuration: 'dev')
+	testmodCompile project(path: ':fabric-events-lifecycle-v0', configuration: 'dev')
```

Comment:
Do note because you have translation keys in your tests, you need to add the resource loader to the testmodCompile

Reply:
oh right.